sys-apps/raspberrypi-sys-mods/raspberrypi-sys-mods-20201026.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit udev systemd
+
+DESCRIPTION="System tweaks for the Raspberry Pi from Raspberry Pi OS"
+
+HOMEPAGE="https://github.com/RPi-Distro/raspberrypi-sys-mods"
+
+SRC_URI="https://archive.raspberrypi.org/debian/pool/main/r/raspberrypi-sys-mods/${PN}_${PV}.tar.xz"
+
+LICENSE="BSD"
+
+SLOT="0"
+
+KEYWORDS="~arm ~arm64"
+
+IUSE=""
+
+RDEPEND="
+	acct-group/input
+	acct-group/i2c
+	acct-group/spi
+	acct-group/gpio
+	acct-group/video
+	"
+
+DEPEND="${RDEPEND}"
```

Comment:
I don't see anywhere the acct-group/* groups used during ebuild? If this is the case, this can be dropped.